Makefile

```diff
@@ -36,7 +36,8 @@ requre:
 
 # Build documentation, prepare man page
 docs: man
-	cd docs && make html
+	make -C docs/ autodocs
+	make -C docs/ html
```

Comment:
Would be also good to update the `clean` target and remove generated docs there.

Reply:
This comment is probably outdated, can you re-review it, please?

Reply:
Yeah, seems covered. Thanks.